discordclient/commands/create_events.go

```diff
@@ -0,0 +1,62 @@
+package commands
+
+import (
+	dgo "github.com/bwmarrin/discordgo"
+)
+
+var createEventsCommand = &dgo.ApplicationCommand{
+	Name:        "create-events",
+	Description: "Create new event",
+}
+
+func createEventsHandler(s *dgo.Session, i *dgo.InteractionCreate) {
+	err := s.InteractionRespond(i.Interaction, &dgo.InteractionResponse{
+		Type: dgo.InteractionResponseModal,
+		Data: &dgo.InteractionResponseData{
+			CustomID: "create_events_" + i.Interaction.Member.User.ID,
```

Comment:
Don't use sting concatenantion, prefere using `fmt.Sprintf`.

Reply:
Thanks, didn't know it existed